src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown

```diff
@@ -0,0 +1,545 @@
+---
+layout: updates/post
+title: "Creating a Web-enabled IoT device with Intel&reg; Edison"
+description: "Internet of Things is really on everyone's lips these
+days, but what if you could create an IoT device which you accessed
+from your web browser? In this article we look at how you can do
+exactly that using Physical Web, Web Bluetooth as well as Node.js,
+running on an Intel&reg; Edison Arduino breakout board."
+published_on: 2016-03-20
+updated_on: 2016-03-20
+authors:
+  - kenchris
+tags:
+  - news
+  - Bluetooth
+  - IoT
+  - WebBluetooth
+  - PhysicalWeb
+  - Edison
+featured_image: /web/updates/images/2016/03/web-enabled-internet-of-things/board.jpg
+---
+
+<p class="intro">
+Internet of Things is really on everyone's lips these days, and
+it makes a tinkerer and a programmer like me very excited. Nothing
+is much cooler than bringing your own inventions to life, and
+being able to talk to them!
+</p>
+
+![Client application](/web/updates/images/2016/03/web-enabled-internet-of-things/tablet-ux.png)
+
+# Web and IoT, a match to be
+
+There are still a lot of hurdles to be overcome before Internet
+of Things will be a huge success. Personally, I am worrying about
+companies requiring people to install their apps for each device
+they produce, cluttering my phone with a multitude of apps that
+I *rarely* use. Something that I would like to avoid.
+
+For this reason, I have been very excited about the
+[Physical Web](https://google.github.io/physical-web/)
+project, which allows devices to broadcast a URL to an online
+website, in a *non-intrusive way*. In combinations with emerging
+web technologies such as
+[Web Bluetooth](https://webbluetoothcg.github.io/web-bluetooth/),
+[Web USB](https://wicg.github.io/webusb/) and
+[Web NFC](https://w3c.github.io/web-nfc/), the site can either
+connect directly to the device or at least explain the proper
+way of doing so.
+
+Web Bluetooth would be a good way of doing so, but in some cases
+you might require a physical connection (say via USB) for security
+reasons, for instance in an office setting.
+
+The web site can also serve as what is called a Progressive Web
+App (aka PWA), which means that the site has a responsive app like
+user experience, works offline and can be added to the home screen.
+Google has a really good explainer about PWAs, which I urge you to
+check out:
+
+[Progressive Web Apps](https://developers.google.com/web/progressive-web-apps)
+
+As a proof of concept, I have been building a small device using
+the Intel&reg; Edison Arduino breakout board. It contains a temperature
+sensor (TMP36) as well as an actuator (colored LED cathode). The
+schematics can be found at the end of this article.
+
+![Breadboard](/web/updates/images/2016/03/web-enabled-internet-of-things/breadboard.jpg)
+
+The Edison is an interesting device as it can run a full Linux
+distribution and therefore allows me to program it easily using
```

Comment:
Suggestion: replace `allows me to program it easily` with `can easily be programmed`